packages/palette/src/elements/BaseTabs/BaseTab.tsx

```diff
@@ -32,31 +30,47 @@ const baseTabActiveMixin = css`
  * or `<BaseTab as={Clickable} onClick={handleClick}>Foo</BaseTab>`
  */
 export const BaseTab = styled(Text)<BaseTabProps>`
-  display: block;
+  display: flex;
   width: 100%;
   position: relative;
   text-decoration: none;
   white-space: nowrap;
   z-index: 1;
-  color: ${color("black60")};
 
-  ${({ active }) => active && baseTabActiveMixin}
-  &:focus,
-  &.active {
-    ${baseTabActiveMixin}
-  }
+  ${(props) => {
+    const states = getThemeConfig(props, { v2: V2_STATES, v3: V3_STATES })
+
+    return css`
+      ${states.default}
+      ${props.hover && states.hover}
+      ${props.focus && states.focus}
+      ${props.active && states.active}
 
-  &:focus {
-    color: ${color("purple100")};
-  }
+      &:hover:not(:disabled) {
+        ${states.hover}
+      }
+
+      &:focus {
+        outline: 0;
+        ${states.focus}
+      }
+    `
+  }}
 `
 
 BaseTab.defaultProps = {
   variant: "mediumText",
   mb: "-1px",
-  pb: 1.5,
-  borderBottom: "2px solid",
-  borderColor: "transparent",
+  alignItems: "flex-start",
+  justifyContent: "center",
+  textAlign: "center",
 }
 
 BaseTab.displayName = "BaseTab"
+
+/**
+ * So this is interesting: We can trick styled-components into correcting the CSS specificity (Text > Clickable)
+ * by simply including a Clickable here.
+ */
+// @ts-ignore
+const __Ignore__ = Clickable
```

Comment:
I'm not 100% sure of the why/how of this. I'm assuming it's because of import order — though having this reference here is also important...